doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -71,6 +71,24 @@ Peripheral samples
   * The CLI command ``fem tx_power_control <tx_power_control>`` replaces ``fem tx_gain <tx_gain>`` .
     This change applies to the sample built with the :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
 
+Matter
+------
+
+  * With the inheritance of Zephyr's :ref:`zephyr:sysbuild` in the |NCS|, some changes are provided to the Matter samples and applications:
+
+    * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` Kconfig option is deprecated and you need to use the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA_GENERATE` Kconfig option instead to enable or disable creating the factory data set during building a Matter sample.
+      To enable factory data support on your device, you still need to set the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA` to ``y``.
+    * Factory data output files are now located in the ``<application_name>/zephyr/`` directory within the build directory.
+    * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_MERGE_WITH_FIRMWARE` Kconfig option is deprecated in sysbuild and you need to use the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA_MERGE_WITH_FIRMWARE` Kconfig option instead to enable or disable merging the factory data HEX file with the final firmware HEX file.
+    * :kconfig:option:`SB_CONFIG_MATTER_OTA` Kconfig option has been added to enable or disable generating Matter OTA package during the building process.
+    * :kconfig:option:`CONFIG_CHIP_OTA_IMAGE_FILE_NAME` Kconfig option is deprecated and you need to use the :kconfig:option:`SB_CONFIG_MATTER_OTA_IMAGE_FILE_NAME` Kconfig option instead to define Matter OTA output filename.
+
+    If you modify any of the factory data-related Kconfig option you need to build a Matter sample with the pristine option to clear the CMake cache.
```

Comment:
```suggestion
    If you modify any of the factory data-related Kconfig options, you need to either build the sample or application with the pristine option to clear the CMake cache or clear the cache manually.
```
Feel free to leave out the manual clearing option if it was left out of the migration guide on purpose.

Reply:
I will leave it because somebody can for example remove the build directory manually and in this way, remove the cache.

Reply:
This can actually go entirely, I tested and if these options are changed, they are picked up and a new image is generated. Must have been an issue that has been fixed sometime without being aware

Reply:
Ok, so I will remove the note from all places.

Reply:
I've removed all those lines that inform about the pristine build.